exe/colorls

```diff
@@ -1,5 +1,9 @@
 #!/usr/bin/env ruby
 
 require 'colorls'
+require 'clocale'
```

Comment:
Can this require be shifted to the lib/colorls.rb and init be shifted to lib/colorls/flags.rb?

Reply:
The rationale was that you normally only want to initialize the locale when running this as a command, but not do this automatically when used as a library.

Reply:
Hmm, but `clocale` gem is anyways being used by the sort method right? (Or, is it used only when `LC_COLLATE=C colorls -1` is given?)

Reply:
Yes, the `strxfrm` is always used, but if `setlocale` is not called, the "C" locale remains active which results in a normal (case-sensitive) string sort (as if setting LC_COLLATE to "C" explicitly).

IMO, when you call a command in your shell, it should respect the locale settings. But if you utilize a library it should not mess with those settings...

Reply:
I thought about this again. Using the Flags class means parsing command line options, so probability is high that colorls was started from a shell. So, I have the initialization to the Flags.process method.